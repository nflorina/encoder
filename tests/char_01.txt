$
D
L
%
M
`
i
X
i
c
<
I
?
:
k
)
+
^
F
v
D
{
&
j
M
+
I
d
k
-
N
Y
@
'
#
H
<
t
'
<
X
~
k
1
END
