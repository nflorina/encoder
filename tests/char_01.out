0 43 0
$$D$$LD$%L$M%$`M$i`$Xi$iX$cD$<c$I<$?I$:?$k:$)k$+)$^+$F^$vF$Dv${L$&{$j&$Mj$+%$I`$dX$kd$-c$N-$YN$@Y$'@$#'$H#$<H$t?$'t$<:$X)$~^$k~
$$D$$LD$%L$M%$`M$i`$Xi$iX$cD$<c$I<$?I$:?$k:$)k$+)$^+$F^$vF$Dv${L$&{$j&$Mj$+%$I`$dX$kd$-c$N-$YN$@Y$'@$#'$H#$<H$t?$'t$<:$X)$~^$k~
